packages/berry-cli/sources/cli.ts

```diff
@@ -38,6 +39,25 @@ function runBinary(path: PortablePath) {
   }
 }
 
+async function printAvailableScripts(configuration: Configuration) {
+  const stdout = process.stdout;
+  const report = await StreamReport.start({configuration, stdout}, async (report: StreamReport) => {
+    const {workspace} = await Project.find(configuration, configuration.startingCwd);
+    if (!workspace)
+      throw new WorkspaceRequiredError(configuration.startingCwd);
+
+
+    var projectCommands: string[] = []
```

Comment:
Just one more question: in yarn v1 we are also printing the commands available from binary scripts:

```
info Commands available from binary scripts: acorn, atob, babylon, browserslist, color-support, commitizen, detect-libc, errno, escodegen, esgenerate, eslint, esparse, esvalidate, flow, git-cz, git-release-notes, gulp, gunzip-maybe, handlebars, import-local-fixture, is-ci, jest, jest-runtime, js-yaml, jsesc, jsinspect, json5, loose-envify, miller-rabin, mkdirp, needle, node-pre-gyp, nopt, oc, opencollective, prettier, rc, regjsparser, rimraf, sane, semver, sha.js, shjs, sshpk-conv, sshpk-sign, sshpk-verify, strip-indent, strip-json-comments, uglifyjs, uuid, watch, webpack, which
```

Wondering if we should do the same in this task? 🤔

Reply:
I'd put it behind a `[-v,--verbose]` option 🤔